source/Weapon.h

```diff
@@ -202,8 +203,6 @@ class Weapon {
 	// Return the ranges at which the weapon's damage dropoff begins and ends.
 	const std::pair<double, double> &DropoffRanges() const;
 
-
-protected:
 	// Legacy support: allow turret outfits with no turn rate to specify a
```

Comment:
These should still be restricted to the Outfit class, use `friend`

Reply:
done